### DISCUSSION TITLE
Can deepmd calculate the Hessian Matrix of PES?

### DISCUSSION THREAD

**Discussion by liangwh534** [deepmodeling/deepmd-kit#771] [Q&A]
I'm using deepmd-1.2.4 and I want to get the Hessian Matrix of a specific configuration , can deepmd do this?
If deepmd can do this, what's the API of this? I want to call the subroutine directly in Python. 
Thanks for help！

**Comment by jameswind** [comment]
Currently deepmd-kit only support the calculation of forces (including
python api:
https://deepmd.readthedocs.io/en/master/use-deepmd-kit.html#model-inference
).
For Hessian you may use finite difference for now ...
